docs/courseInstance.md

```diff
@@ -112,10 +120,38 @@ Some instructors may wish to hide their course from the list of available course
 }
 ```
 
-Note that _this is not a security setting_. Students may still enroll in the course instance if they get access to the URL, either through friends or by [Forced Browsing Attacks](https://owasp.org/www-community/attacks/Forced_browsing). Instructors that wish to actually restrict course enrollment to a specific list of students should instead use well-defined access rules with restrictions by UIDs, Institution, or through LTI support.
+Note that _this is not a security setting_. Students may still enroll in the course instance if they get access to the URL, either through friends or by [forced browsing attacks](https://owasp.org/www-community/attacks/Forced_browsing). Instructors that wish to actually restrict course enrollment to a specific list of students should instead use well-defined access rules with restrictions by institution or UIDs, described under ["Access controls"](#access-controls).
```

Comment:
I think the "forced browsing attacks" sounds scary and isn't really adding anything of value here. I also think we can link directly to the UID access control section, which is what I think is relevant here?
```suggestion
Note that _this is not a security setting_. Students may still enroll in the course instance if they get access to the URL, such as from a friend. Instructors that wish to actually restrict course enrollment to a specific list of students should instead use an access rules with an explicit list of UIDs, as described under ["Controlling access by UID"](#controlling-access-by-uid).
```